xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-test/xwiki-platform-administration-test-pageobjects/src/main/java/org/xwiki/administration/test/po/GroupsPage.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.xwiki.administration.test.po;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.xwiki.test.ui.po.BasePage;
+import org.xwiki.test.ui.po.LiveTableElement;
+
+/**
+ * Page Object for the Administration/Groups page.
+ * 
+ * @version $Id$
+ * @since 8.1M2
+ */
+public class GroupsPage extends BasePage
+{
+    LiveTableElement groupsLiveTable = new LiveTableElement("groupstable");
+
+    @FindBy(id = "addNewGroup")
+    private WebElement addGroupButton;
+
+    @FindBy(css = "#addnewgroup .button.create")
+    private WebElement createGroupButton;
+
+    public void clickAddNewGroupButton()
+    {
+        this.addGroupButton.click();
+    }
+
+    public void clickCreateGroupButton()
+    {
+        this.createGroupButton.click();
+    }
+
+    // Method to create a new Group
```

Comment:
You need to use multiline javadoc comments (/** **/) and not inline code comments (//).